docs/run-node/interact-node.md

```diff
@@ -52,12 +52,13 @@ You should see two delegations, the first one made from the `gentx`, and the sec
 
 The Protobuf ecosystem developed tools for different use cases, including code-generation from `*.proto` files into various languages. These tools allow to build clients easily. Often, the client connection (i.e. the transport) can be plugged and replaced very easily. Let's explore one of the most popular transport: [gRPC](../core/grpc_rest.md).
 
-Since the code generation library largely depends on your own tech stack, we will only present two alternatives:
+Since the code generation library largely depends on your own tech stack, we will only present three alternatives:
 
 - `grpcurl` for generic debugging and testing,
+- programmatically via Go,
 - CosmJS for JavaScript/TypeScript developers.
 
-### grpcurl: Reflection, Queries, and Simulation
+### grpcurl
 
 [grpcurl])https://github.com/fullstorydev/grpcurl is like `curl` but for gRPC. It is also available as a Go library, but we will use it only as a CLI command for debugging and testing purposes. Follow the instructions in the previous link to install it.
```

Comment:
No need to apply this here but we could prob. open a PR with doc improvements / minor fixes to the SDK:
```suggestion
[grpcurl](https://github.com/fullstorydev/grpcurl) is like `curl` but for gRPC. It is also available as a Go library, but we will use it only as a CLI command for debugging and testing purposes. Follow the instructions in the previous link to install it.
```